consolidate/BC/BoundaryCondition.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Jul 25 11:38:36 2020
+
+@author: andre
+"""
+
+import numpy as np
+
+class BoundaryCondition():
+    
+    def __init__(self, deck,geometry,meshes):
+        self.deck = deck
+        self.geometry = geometry
+        self.meshes=meshes
+        self.set_temperatures()
+        self.set_conductivity()
+        self.set_density()
+        self.set_specific_heat()
+        self.set_diffusivity()
+        self.set_viscosity()
+        self.set_dic()
+        self.set_heat_input_density()
+        
+    def set_temperatures(self):    
+        
+        T=    {
+        "Top Adherent":np.zeros((self.meshes.ny.get("Top Adherent"), self.meshes.nx)) ,
+        "HE"       :np.zeros((self.meshes.ny.get("HE"), self.meshes.nx)) ,
+        "Bottom Adherent":np.zeros((self.meshes.ny.get("Bottom Adherent"), self.meshes.nx)) 
+                  }
```

Comment:
Maybe you could create a variable or something else to avoid writing this several times